classes/book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './items'
+
+class Book < Item
+  attr_accessor :archived
+
+  def initialize(publisher, cover_state, label, publish_date)
+    super
+    @publisher = publisher
+    @cover_state = cover_state
+  end
+
+  def can_be_archived?()
+    archived = true || cover_state = 'bad'
+  end
```

Comment:
- [ ] Kindly consider can_be_archived?() in the Book class should override the method from the parent class and should return true if the parent's method returns true OR if cover_state equals to "bad" ,so it will be like this
 ```ruby
   def can_be_archived?
      super || @cover_state = 'bad'
    end
```

Reply:
This change has been done well